### PR TITLE
feat(AAiT-web-1): Implement Teams Page

### PR DESCRIPTION
**Pull request by PaulosNed**
I have added the teas-page. It has the hero section as well as the members grid section. I have created a card component for each member to be mapped into and I have also modified my global css to add custom classes with defined property.

Screenshot (6)
Screenshot (7)
Screenshot (8)
Screenshot (9)

### CODE REVIEW

File: aait/web/group-1/tailwind.config.ts [hunk 1]
```diff
@@ -1,20 +1,23 @@
-import type { Config } from 'tailwindcss'
+import type { Config } from "tailwindcss";
 
 const config: Config = {
   content: [
-    './pages/**/*.{js,ts,jsx,tsx,mdx}',
-    './components/**/*.{js,ts,jsx,tsx,mdx}',
-    './app/**/*.{js,ts,jsx,tsx,mdx}',
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```
Comment (jebessaD) [inline aait/web/group-1/tailwind.config.ts:5]: Here, you've made unnecessary changes. This might make more changes happen when others add their own changes

File: aait/web/group-1/public/images/facebook.svg
Comment (jebessaD) [inline aait/web/group-1/public/images/facebook.svg:1]: User React icon instead of svg

**Review by jebessaD — APPROVED** [review]

File: aait/web/group-1/tailwind.config.ts (hunk 1 above)
Comment (PaulosNed) [reply aait/web/group-1/tailwind.config.ts]: I have reverted to the previous version
